src/sailjail.c

```diff
@@ -361,6 +361,11 @@ int main(int argc, char* argv[])
     ok = g_option_context_parse(options, &argc, &argv, &error);
 
     if (ok) {
+        /* Remove "--" from the list if it is there */
+        if (argc > 1 && !strcmp(argv[1], "--")) {
+            argv++;
+            argc--;
```

Comment:
Maybe I'm missing something but this does not look right: skip argv[0] If argv[1] has "--" so that [foo, --, bar] becomes [--, bar]? And we then try to exec "--" below?

Reply:
Heh, thanks for spotting this! argv[0] is not used for anything as it won't try to exec argv[0] but argv[1] instead. So I guess it doesn't matter much here, but you are right that this is not doing what it says it is. I'll fix it so it actually removes that "--" argument.